question-service/app/routers/questions.py

```diff
@@ -80,4 +107,12 @@ async def batch_upload(questions: List[CreateQuestionModel]):
         result = await batch_create_questions(questions)
         return result
     except BatchUploadFailedError as e:
-        raise HTTPException(status_code=400, detail=str(e))
\ No newline at end of file
+        raise HTTPException(status_code=400, detail=str(e))
+
+@router.get("/enum/category", response_description="Get all valid question categories", response_model=List[CategoryEnum])
+async def get_categories():
```

Comment:
Is there a need to make this `async` since we not awaiting on any functions?

Reply:
The [FastAPI documentation](https://fastapi.tiangolo.com/async/) seems to recommend usage of `async` in this case for some reason.